modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractPhpCodegen.java

```diff
@@ -68,7 +79,7 @@ public AbstractPhpCodegen() {
         apiPackage = invokerPackage + "\\" + apiDirName;
         modelPackage = invokerPackage + "\\" + modelDirName;
 
-        setReservedWordsLowerCase(
+        super.registerReservedWordsCaseInsensitive(
                 Arrays.asList(
                         // local variables used in api methods (endpoints)
```

Comment:
It's absolutely correct, that reserved words("abstract", "and", "array", "as", "break", "callable", "case", "catch", "class") in PHP are case insensitive. However variables aren't. You can check it with simple command:
```bash
$ php -r '$httpBody = "FooBar"; $httpbody = "foobar"; echo $httpBody . PHP_EOL; echo $httpbody;'
// FooBar
// foobar
```
Just keep it in mind, I wouldn't say that it breaks something, but we should be precise in syntax language related topics. Anyway, it's good to warn people not to use `$httpbody` variable when generator contains `$httpBody` to avoid confusion while it's 100% acceptable for PHP parser.